src/app/components/calendar.js

```diff
@@ -0,0 +1,78 @@
+import axios from 'axios';
+
+export default function (container) {
+    const active = 'active';
+    const boxes = [...container.querySelectorAll('.js-box')];
+    const boxIcons = [...container.querySelectorAll('.js-icon-box')];
+    const cross = 'js-cross';
```

Comment:
name of variables, read this, please
https://github.com/kettanaito/naming-cheatsheet